resources/dicts/patrols/general/medcat.json

```
@@ -479,147 +838,345 @@
+                "can_have_stat": ["p_l"],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["app1"],
                         "mutual": false,
                         "values": ["respect", "trust"],
                         "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "At first app1 doesn't speak up, however p_l notices the apprentice being distracted, and gently asked what was wrong. app1 admitted to the vision, and the two had a discussion of the clair_list app1 felt in the dream, leaving app1 feeling relieved.",
+                "exp": 20,
+                "weight": 20,
+                "herbs": ["random_herbs"],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["app1"],
+                        "mutual": true,
+                        "values": ["respect", "trust"],
+                        "amount": 5
                     },
                     {
                         "cats_to": ["p_l"],
-                        "cats_from": ["r_c"],
+                        "cats_from": ["app1"],
                         "mutual": true,
-                        "values": ["jealous"],
+                        "values": ["jealousy"],
                         "amount": -5
                     }
                 ]
             }
         ],
         "fail_outcomes": [
             {
-                "text": "p_l shakes {PRONOUN/p_l/poss} head sadly. {PRONOUN/p_l/subject/CAP} can't make sense of the vision.",
+                "text": "p_l shakes {PRONOUN/p_l/poss} head sadly as app1 describes the dream containing flashes of clair_list. p_l can't make sense of the vision.",
                 "exp": 0,
                 "weight": 20,
                 "relationships": [
                     {
                         "cats_to": ["p_l"],
-                        "cats_from": ["r_c"],
+                        "cats_from": ["app1"],
                         "mutual": true,
                         "values": ["respect", "trust"],
                         "amount": -5
-                    },
-                    {
-                        "cats_to": ["p_l"],
-                        "cats_from": ["r_c"],
-                        "mutual": false,
-                        "values": ["jealous"],
-                        "amount": 5
                     }
                 ]
             },
             {
-                "text": "s_c watches r_c with undisguised envy, desperately wishing {PRONOUN/s_c/subject} {VERB/s_c/were/was} the one to receive the vision. Looking back nervously, r_c withdraws from the group, dismissing the dream as unimportant.",
+                "text": "app1 doesn't tell p_l about the dream, keeping it to {PRONOUN/app1/self}. p_l doesn't need to know that app1 saw flashes of clair_list... right?",
```

```suggestion
                "text": "app1 doesn't tell p_l about the dream, keeping it to {PRONOUN/app1/self}. p_l doesn't need to know that app1 saw flashes of prophecy_list_sight... right?",
```
if we specifically want a list of things they could have "seen" in the dream, then prophecy_list_sight will provide more "accurate" options